reactive/archive_auth_mirror.py

```diff
@@ -58,7 +58,17 @@ def config_not_set():
         'blocked', 'Not all required configs set, mirroring is disabled')
 
 
-def configure_static_serve():
+@when('leadership.is_leader')
+def install_cron():
+    cron.install_crontab()
+
+
+@when_not('leadership.is_leader')
+def remove_cron():
+    cron.remove_crontab()
```

Comment:
:thinking: I'd have thought you'd want an extra state for something like `crontab.installed` and then have:

`if leader and not crontab.installed:
   install_crontab() # sets crontab.installed = True
elif not leader and crontab.installed:
   remove_crontab() # sets crontab.installed = False
`

- having that scheme means you won't be stating a file for every Juju hook